Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN make build
 # Final layer: ubuntu with chainlink binary
 FROM ubuntu:16.04
```

Comment:
Unrelated to PR. Our final image size has ballooned to ~370MB. Should we use `alpine` linux instead to get smaller images?